docs/src/examples/components/Label/Variations/LabelExampleOnIconClick.shorthand.tsx

```diff
@@ -1,31 +1,29 @@
 import React from 'react'
 import { Label } from '@stardust-ui/react'
 
-class LabelExampleIconShorthand extends React.Component<{}, { display: string }> {
-  constructor(props) {
-    super(props)
-    this.state = {
-      display: 'inline-block',
-    }
-  }
+class LabelExampleOnIconClickShorthand extends React.Component {
+  state = { hidden: false }
 
-  public hide = () => {
-    this.setState({ display: 'none' })
+  hide = () => {
+    this.setState({ hidden: true })
+    setTimeout(() => this.setState({ hidden: false }), 2000)
   }
 
-  public render() {
-    const { display } = this.state
+  render() {
+    const { hidden } = this.state
+
+    if (hidden) return 'Returning in 2 seconds...'
+
     return (
       <Label
         circular
-        style={{ display }}
+        content="Removable label"
```

Comment:
Do not use inline styles in examples.  Our components and examples need to be exemplar, showing users how to accomplish tasks without inline styles.  Using inline styles defeats one of the core purposes of Stardust.